ctrlf.el

```diff
@@ -467,7 +575,7 @@ When doing regexp search, wrap it with \"\\_<\" and \"\\_>\". When there's no
 symbol at point, return nil."
   (let ((symbol (thing-at-point 'symbol t)))
     (when symbol
-      (if ctrlf--regexp-p
+      (if (memq ctrlf--style '(regexp fuzzy-regexp))
```

Comment:
Let's just get rid of this `\\_<` logic. It breaks the abstraction very badly, as you point out. The better way to handle this, I think, would be to have a new search style that searches for a single word only. That translator function would automatically wrap the search query inside `\\_<` and `\\_>`.

Reply:
Done.

Reply:
Seems reasonable. Might be a little tricky to do without making things complicated, but I think it's doable. Good idea.